_posts/2019-07-22-aws-event-bridge.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title: AWS Event Bridge Introduced
+date: 2019-03-29
+tags: aws event-bridge
+author: JK Gunnink
+---
+
+Earlier this month Amazon Web Services introduced a new service to their suite called EventBridge,
+and amongst the community it's created quite a buzz. [Some are even claiming it to be the biggest
+thing since Lambda](https://www.trek10.com/blog/amazon-eventbridge/).
+
+## What is it?
+
+According to the EventBridge landing page:
+
+> Amazon EventBridge is a serverless event bus that makes it easy to connect applications together
+> using data from your own applications, Software-as-a-Service (SaaS) applications, and AWS
+> services. EventBridge delivers a stream of real-time data from event sources, such as Zendesk,
+> Datadog, or Pagerduty, and routes that data to targets like AWS Lambda. You can set up routing
+> rules to determine where to send your data to build application architectures that react in real
+> time to all of your data sources. EventBridge makes it easy to build event-driven applications
+> because it takes care of event ingestion and delivery, security, authorization, and error handling
+> for you.
+
+Building on the CloudWatch service which can be used a an event trigger, EventBridge allows events
+from AWS and external services to be captured by EventBridge and forwarded on to a destination that
+fits the event's use case based on it's contents.
+
+Some architectures and patterns come to mind when dealing with events, one such example is the
+rapids, rivers and ponds architecture. EventBridge allows you to filter events from many sources, at
+potentially high volume (rapids), and based on the event forward it through to the next service in
+it's lifecycle (rivers) before it finishes at it's desitination. As a comparison, a current
+implementation might be to recieve an event from CloudWatch which is sent to a lambda, this is then
```

Comment:
The current approach would be to use SNS, with SNS Filters.  
To integrate with CloudWatch Events, it requires an event dispatcher to forward events from CloudWatch to SNS - which adds latency and additional cost.

Reply:
resolved by 817503b